pages/common/standard.md

```diff
@@ -0,0 +1,31 @@
+# standard
+
+> The JavaScript Standard Style tool for linting and fixing JavaScript code.
+
+- Lint all JavaScript source files in the current directory:
+
+`standard`
+
+- Lint specific JavaScript files:
```

Comment:
If the command allows multiple files, change the example to `file(s)`, if not, change the description to `file`.

Reply:
:+1: I'll change it to `file(s)`.

Reply:
Awesome, I was touched by your rigorous attitude. :+1:  @agnivade